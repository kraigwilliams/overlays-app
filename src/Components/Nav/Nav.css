nav,.nav-color{
    margin-top:none;
    background:#1E555C;
    margin-bottom:30px;
    color:white;
}

.nav-color:hover{
color:#F15152;
}