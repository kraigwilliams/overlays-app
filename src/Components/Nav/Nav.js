import React ,{Component} from 'react';


import {Link} from 'react-router-dom';
import './Nav.css'

/* 
Nav component shows the Login/Logout functionality at the top of every page.
*/
class Nav extends Component{


    
    render(){
        return(
            <nav>

 <Link className="nav-color" to="/logout">Login / Logout</Link>

     </nav>
    )




}
}

export default Nav