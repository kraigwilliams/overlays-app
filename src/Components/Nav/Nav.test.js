import React from 'react';
//import { render } from '@testing-library/react';
import Nav from './Nav'
import {BrowserRouter} from 'react-router-dom'
import ReactDOM from  'react-dom';
import renderer from 'react-test-renderer';

describe('<Nav/>',()=>{
  it ('is able to render',()=>{
    const div = document.createElement('div')
    ReactDOM.render(<BrowserRouter><Nav/></BrowserRouter>, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<BrowserRouter><Nav/></BrowserRouter>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });

  
})