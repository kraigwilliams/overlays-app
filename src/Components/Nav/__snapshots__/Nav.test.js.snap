// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Nav/> renders the UI as expected 1`] = `
<a
  href="/logout"
  id="nav-color"
  onClick={[Function]}
>
  Login / Logout
</a>
`;
